src/interactive/modules.ts

```diff
@@ -91,16 +93,24 @@ async function updateModuleForEditor(editor: vscode.TextEditor) {
     try {
         mod = await getModuleForEditor(editor)
     } catch (err) {
-        console.error(err)
-        return
+        if (g_languageClient) {
+            telemetry.traceTrace({
+                message: err.toString(),
+                severity: SeverityLevel.Error
+            })
+        }
     }
 
     let loaded = false
     try {
         loaded = await g_connection.sendRequest(requestTypeIsModuleLoaded, mod)
     } catch (err) {
-        console.error(err)
-        return
+        if (g_connection) {
+            telemetry.traceTrace({
+                message: err.toString(),
+                severity: SeverityLevel.Error
+            })
+        }
```

Comment:
Same here, I think this should show up as an exception, not a trace, and we should add a higher level error handling like in the other PR to catch such errors.